CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [15.1.2](https://github.com/theforeman/puppet-foreman/tree/15.1.2) (2020-11-11)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman/compare/15.1.1...15.1.2)
+
+**Breaking changes:**
```

Comment:
Perhaps we can make it non-breaking: https://github.com/theforeman/puppet-foreman/pull/902/files#r522016518

Reply:
Rebased.